Update: Journey -> ladder (#3726)
dfinity/portal: docs/tutorials/developer-journey/level-0/06-intro-dfx.mdx

When no flags are used, the `dfx new` command will create a new project using the default Motoko template. To create a project using the Rust project template, the flag `--type=rust` should be included in the command.

> In this developer ladder, you will be using Motoko for our development language, so you do not need to pass any additional flags with this command.

Author: ```suggestion
In this tutorial series, you will be using Motoko for the development language, so you do not need to pass any additional flags with this command.
```
